### DISCUSSION TITLE
PostgreSQL - Wrong Password (Right Password)

### DISCUSSION THREAD

**Discussion by mphil-ip** [Budibase/budibase#4336] [Help]
Database: hosed on DO
Budibase: hosted on DO

Hello everyone! New to Budibase and very excited to see what it has to offer. 

I am experiencing an issue where I try to connect a Postgres database to an app and no matter what credentials I feed it I am always getting an error that the password is wrong. I know this is impossible because I comfortably connected to the DB from other sources including Retool. 

Additionally, the IP address of the droplet is whitelisted on the DB cluster. 

Anyone have any thoughts? 

Thank you in advance!

**Comment by SantoshSrinivas79** [comment]
Hi @mpolj ... I have Budibase self-hosted and Postgress running as a local service. Works without issues.
Here is my setup. Pls post your screenshots.

<img width="670" alt="Screenshot 2022-02-05 at 9 15 54 AM" src="https://user-images.githubusercontent.com/1036163/152627635-f53cc5fb-0387-4b01-92b5-d27d09fafbba.png">
